bindata/v3.11.0/openshift-apiserver/audit-policies-cm.yaml

```diff
@@ -0,0 +1,131 @@
+apiVersion: v1
```

Comment:
am not a fan of duplication of these files. They should come from library-go. All apiservers share them.

Reply:
yes, working on a follow up library-go pr

Reply:
opened https://github.com/openshift/library-go/pull/858 PTAL